data/json/professions.json

```
@@ -916,6 +916,95 @@
       "female": [ "panties", "sports_bra" ]
     }
   },
+  {
+    "type": "profession",
+    "id": "professional_diver",
+    "name": "Professional Diver",
+    "description": "You spent a large part of your time exploring lakes and diving in the New England region, giving you lots of knowledge about the waters in the local area.",
+    "points": 3,
+    "skills": [ { "level": 4, "name": "swimming" }, { "level": 1, "name": "survival" } ],
+    "items": {
+      "both": {
+        "items": [ "wetsuit", "wetsuit_hood", "dive_bag", "diving_watch", "goggles_swim", "scuba_tank", "swim_fins" ],
+        "entries": [ { "group": "charged_flashlight" } ]
+      }
+    }
+  },
+  {
+    "type": "profession",
+    "id": "underwater_welder",
+    "name": "Underwater Welder",
+    "description": "Through years of underwater welding, you've honed a deep understanding of submerged structures in the New England region.",
+    "points": 3,
+    "skills": [ { "level": 3, "name": "swimming" }, { "level": 2, "name": "fabrication" }, { "level": 2, "name": "mechanics" } ],
+    "items": {
+      "both": {
+        "items": [ "wetsuit", "wetsuit_hood", "dive_bag", "diving_watch", "goggles_swim", "small_scuba_tank", "swim_fins" ],
```

Since welding goggles would compete more with swim goggles and welder will need a welding tool.
```suggestion
    "items": [
      "wetsuit",
      "wetsuit_hood",
      "welding_mask_raised",
      "dive_bag",
      "diving_watch",
      "goggles_swim",
      "small_scuba_tank",
      "swim_fins"
    ],
```